### PR TITLE
Bye bye bower pt1

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

Begins removing bower from the app, to use only npm. In this case, from `static/src/javascripts` and `static/public/javascripts`.
## What is the value of this and can you measure success?

simplification and hopefully more immediately apprendable codebase
## Does this affect other platforms - Amp, Apps, etc?

no
## Request for comment

@rich-nguyen

### CODE REVIEW

**Comment by TBonnin** [comment]
screen shot 2016-06-22 at 14 36 28 💃 💃 💃

File: grunt-configs/copy.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ module.exports = function (grunt, options) {
                 },
                 {
                     expand: true,
+                    cwd: 'node_modules',
+                    src: [
+                        'react/dist/react-with-addons.js',
+                        'JSON/json2.js'
+                    ],
```
Comment (rich-nguyen) [inline grunt-configs/copy.js]: would it be possible to locate all the vendor based dependencies from npm? then we could remove `static/src/javascripts/vendor`, and have `formstack`, `prebid` and `stripe` installed from `node_modules`

**Comment by rich-nguyen** [comment]
“apprendable”, the neologist’s solution to a readable codebase born from an unreadable pr (due to github crashing). 👍  Great stuff

File: grunt-configs/copy.js (hunk 1 above)
Comment (sndrs) [inline grunt-configs/copy.js]: yeah very happy to do that in subsequent PRs, was trying to limit this to just changes that fell out of the first bower going

**Comment by jamespamplin** [comment]
:tada: thanks @sndrs

**Comment by kaelig** [comment]
w00t

**Comment by sndrs** [comment]
this is too big i think, and doesn't help us move on. closing for now, will implement it even more piecemeal as we move away from require

**Pull request closed without merging**